docs/en/operation/dataset_auto_scaling.md

```diff
@@ -0,0 +1,424 @@
+# Demo - Dataset Customized Elastic Scaling
```

Comment:
I think "Cache Runtime Auto Scaling" or maybe "Dataset Auto Scaling" is okay. "Customized Elastic Scaling` seems a little bit weird to me.